public/js/hiringPartner.js

```diff
@@ -145,47 +145,28 @@ async function getHiringPartners () {
             'Content-Type': 'application/json',
         },
     })
-    .then( hiringPartnerInfo => hiringPartnerInfo.json())
+    .then(hiringPartnerInfo => hiringPartnerInfo.json())
     .then(hiringPartnerInfo => displayHiringPartnerHandler(hiringPartnerInfo.data))
 }
 
+getHiringPartners()
+
+
 /**
  * Runs a foreach through each hiring partner object and outputs HTML elements with hiring partner's details
  *
  * @param partnerCompanies is an array of objects which contains information about hiring partners
  */
 function displayHiringPartnerHandler(partnerCompanies){
-    let companyDisplayer = document.getElementById('companies')
-    let companyInformation = ''
+    let companyNamesHTML = '';
+    console.log('hello')
```

Comment:
I'm not sure we should be console logging hello on master

Reply:
Done